samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52820.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&radio {
+	status = "okay";
+	/* This is an example number of antennas that may be available
```

Comment:
We are not going to use configuration for antenna matrix designed by Nordic here?

Reply:
Also Rx instead of Tx

Reply:
Also `antenna switching when AoA is enabled` ?

Reply:
The whole file content is wrong. I've changed it to proper one.